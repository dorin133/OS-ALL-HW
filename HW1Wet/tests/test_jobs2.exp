smash> smash> smash> [1] sleep 10& : 111003 0 secs
smash> smash> [1] sleep 10& : 111003 0 secs
[2] sleep 20& : 111004 0 secs
smash> smash> smash> [1] sleep 10& : 111003 0 secs
[2] sleep 20& : 111004 0 secs
[3] sleep 30& : 111005 0 secs
[4] sleep 40& : 111006 0 secs
smash> smash: sending SIGKILL signal to 4 jobs:
111003: sleep 10&
111004: sleep 20&
111005: sleep 30&
111006: sleep 40&
