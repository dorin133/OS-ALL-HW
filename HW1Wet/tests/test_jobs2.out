smash> smash> smash> [1] sleep 10& : 111003 0 secs
smash> smash> [1] sleep 10& : 111003 0 secs
[2] sleep 20& : 111005 0 secs
smash> smash> smash> [1] sleep 10& : 111003 0 secs
[2] sleep 20& : 111005 0 secs
[3] sleep 30& : 111009 0 secs
[4] sleep 40& : 111010 0 secs
smash> smash: sending SIGKILL signal to 4 jobs:
111003: sleep 10&
111005: sleep 20&
111009: sleep 30&
111010: sleep 40&
